Automattic/wp-calypso
# Agency Dashboard: Show the Dashboard item in the navigation menu after agency signs up

#### Proposed Changes

If potential agency partner tries to register using the self-serve signup form, they won't see the `Dashboard` item in the navigation when the form is submitted until they refresh the page. This PR fixes that issue (see the Asana card for screenshots).

#### Testing Instructions



- Checkout this branch locally
- Run yarn `start-jetpack-cloud`
- Create a new WPCOM account (see [hint](2e233-pb/raw/))
- Verify your email address
- Visit http://jetpack.cloud.localhost:3000/agency/signup and complete the signup form
- Verify that once the form is submitted, you will see the `Dashboard` item in the navigation menu
- Refresh the page and verify that you see the same navigation items after the refresh


#### Pre-merge Checklist






Related to 1202619025189113-as-1203155829030695
